hi-ml-cpath/src/health_cpath/utils/analysis_plot_utils.py

```diff
@@ -190,35 +190,35 @@ def _plot_crossval_roc_and_pr_curves(crossval_dfs: Dict[int, pd.DataFrame], roc_
         # assert len(non_unique_slides) == 0
         scores = tile_scores.first()
 
-        plot_roc_curve(labels, scores, legend_label=f"Fold {k}", ax=roc_ax)
-        plot_pr_curve(labels, scores, legend_label=f"Fold {k}", ax=pr_ax)
+        plot_roc_curve(labels, scores, legend_label=f"Child {k}", ax=roc_ax)
+        plot_pr_curve(labels, scores, legend_label=f"Child {k}", ax=pr_ax)
     legend_kwargs = dict(edgecolor='none', fontsize='small')
```

Comment:
not sure why codecov complains here. Is this not included in tests?

Reply:
It doesn't appear that we have tests for these functions. Might be out of scope of this PR?

Reply:
ok, resolving this now. Perhaps open a ticket for the tests?